.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,20 +1,39 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
 <!-- Please, prefix the report title with the language it applies to within brackets, such as [java] or [apex].
 If not specific to a language, you can use [core]. -->
 
-<!-- NB: issues about the rule designer should be opened at https://github.com/pmd/pmd-designer/issues -->
-
 **Affects PMD Version:** 
 
-**Rule:**
+Make sure, to test with the latest PMD version.
 
 **Description:**
 
+A clear and concise description of what the bug is.
+
+**Exception Stacktrace:**
+
+```
+# Copy-paste the stack trace here
+```
+
 **Code Sample demonstrating the issue:**
 
 ```
 
 ```
 
+**Steps to reproduce:**
+
+1.  Compile the project: `mvn verify`
```

Comment:
I'm not sure what this means. Is this listing a template, that users should modify? Is the mvn command referring to building a user project, or pmd itself?

Maybe we should encourage users to provide a MWE

Reply:
That was the goal - these are the steps for us to build their MWE. Maybe that's not clear... 

And yes - every template needs to be modified... it's just a template :smile: